src/runtime/vm/profiler/vm.h

```diff
@@ -51,7 +51,7 @@ class VirtualMachineDebug : public VirtualMachine {
                     const std::vector<ObjectRef>& args) final;
 
   std::unordered_map<Index, std::string> packed_index_map_;
-  std::unordered_map<Index, std::vector<double>> op_durations_;
+  std::unordered_map<Index, std::vector<TypedPackedFunc<int64_t()>>> op_durations_;
```

Comment:
this isn't really durations anymore, is it?

Reply:
I mean, it is. You just have to call the function to get the duration.